Make.config

```diff
@@ -30,7 +30,7 @@ PACKAGE_HEAD_REV=$(shell git rev-parse HEAD)
 # on wrench, because wrench technically builds hashes, not branches)
 #
 # 
-PACKAGE_HEAD_BRANCH=master
+PACKAGE_HEAD_BRANCH=mono-2017-06
```

Comment:
This change must not be merged to master.

Reply:
Let me fix that

Reply:
The value has been reverted to `master`